onnxruntime/core/providers/cpu/tensor/dynamicquantizelinear.cc

```diff
@@ -66,7 +69,11 @@ Status DynamicQuantizeLinear<T>::Compute(OpKernelContext* ctx) const {
 
   // quantize the data
   auto* output = y.template MutableData<T>();
-  MlasQuantizeLinear(x_data, output, num_of_elements, scale, zero_point);
+
+  concurrency::ThreadPool* tp = ctx->GetOperatorThreadPool();
```

Comment:
Why not also parallel above two:
  auto min = ConstEigenVectorMap<float>(x_data, num_of_elements).minCoeff();
  auto max = ConstEigenVectorMap<float>(x_data, num_of_elements).maxCoeff();

Reply:
will add SIMD for the min/max reduce in a separate PR.